File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        HashMap<Integer, Integer> frequency = new HashMap<>();

        for(Integer num: nums){
            frequency.put(num, frequency.getOrDefault(num, 0) + 1);
        }

        List<Integer> sortedKeys = new ArrayList<>(frequency.keySet());

        Collections.sort(sortedKeys, (a, b) -> frequency.get(b) - frequency.get(a));

        int[] answer = new int[k];

        for(int i = 0; i < k; i++){
            answer[i] = sortedKeys.get(i);

        }

        return answer;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        streak = 0
        mx = 0

        for num in nums:
            if num == 1:
                streak += 1
            else:
                mx = max(mx, streak)
                streak = 0

        return max(mx, streak)